src/palimpzest/query/operators/critique_and_refine_convert.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import annotations
+
+from typing import Any
+
+from palimpzest.constants import MODEL_CARDS, Model, PromptStrategy
+from palimpzest.core.data.dataclasses import GenerationStats, OperatorCostEstimates
+from palimpzest.core.elements.records import DataRecord
+from palimpzest.query.generators.generators import generator_factory
+from palimpzest.query.operators.convert import LLMConvert
+
+# TYPE DEFINITIONS
+FieldName = str
+
+
+class CriticConvert(LLMConvert):
```

Comment:
nit: can we rename this to CriticAndRefineConvert?

Reply:
Agreed, that is cleaner